### PR TITLE
Instruction Correction

### PR DESCRIPTION
**Pull request by joinimran**
Now we can manage GDPR data subject requests and i have updated the document.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/786

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/search-for-and-find-personal-data.md [hunk 1]
```diff
@@ -27,9 +27,7 @@ Article 4 – Definitions
...
-
-So, while you can’t currently use Content Search to find personal data at rest in Exchange Online mailboxes, you can use the sensitive information types you curate for GDPR to find and protect personal information as it is sent through email.
+Finding personal data subject to GDPR relies on using sensitive information types in Office 365. These define how the automated process recognizes specific information types such as health service numbers and credit card numbers. You can use data loss prevention policies to find personal data in mail while in transit. You can use the sensitive information types you curate for GDPR to find and protect personal information as it is sent through email. Also see [managed GDPR data subject requests with the DSR case tool in the security and compliance center](https://docs.microsoft.com/microsoft-365/compliance/manage-gdpr-data-subject-requests-with-the-dsr-case-tool).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-for-and-find-personal-data.md]:
```suggestion
Finding personal data subject to GDPR relies on using sensitive information types in Office 365. These define how the automated process recognizes specific information types such as health service numbers and credit card numbers. You can use data loss prevention policies to find personal data in mail while in transit. You can use the sensitive information types you curate for GDPR to find and protect personal information as it is sent through email. Also see [Managed GDPR data subject requests with the DSR case tool in the Security & Compliance Center](https://docs.microsoft.com/microsoft-365/compliance/manage-gdpr-data-subject-requests-with-the-dsr-case-tool).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@BrendaCarter - PR has been copy edited and is ready for final review. This is based on issue #786.

Thank you.

cc: @kenwith

**Comment by LauraWi** [comment]
@markjjo This touches on Content Search and DSR requests -- can you validate?

**Comment by markjjo** [comment]
@LauraWi PR is technically accurate; merged

**Pull request merged**